Sources/Commands/SwiftPackageTool.swift

```diff
@@ -587,27 +587,30 @@ extension SwiftPackageTool {
 
         @Option(help: "The revision to edit", transform: { Revision(identifier: $0) })
         var revision: Revision?
-        
+
         @Option(name: .customLong("branch"), help: "The branch to create")
         var checkoutBranch: String?
-        
+
         @Option(help: "Create or use the checkout at this path")
         var path: AbsolutePath?
-        
+
         @Argument(help: "The name of the package to edit")
         var packageName: String
 
         func run(_ swiftTool: SwiftTool) throws {
+            swiftTool.diagnostics.emit(.warning("Editing dependencies can be done via local overrides. 'edit/unedit' are no longer needed and will be deprecated soon."))
```

Comment:
Maybe there is a better way to describe this? When I [search for “swiftpm local overrides”](https://www.google.com/search?q=swiftpm+local+overrides), the first hit is instructions for Xcode, a few hits are instructions to use this `edit` command, and the remainder are irrelevant.

Even after searching, I honestly don’t know how to do this on Linux without using `edit`.

Reply:
It's something we're going to implement as part of explicit overrides which are needed for the package registry support.

Reply:
Probably the warning will have to come once there's a better implementation existing in SwiftPM CLI

Reply:
this is possible even without the introduction of explicit overrides by pointing the Package.swift to a local path instead of the URL, but I am happy to hold on to this PR to figure out of we want to do the explicit overrides feature first.

Reply:
Ah, I see — adding a path dependency in your root package.  In that case the warning could be added now (I was thinking that this was only an Xcode thing).